app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -11,7 +11,7 @@
 <% else %>
   <%= text_f f, :datastore, :class => "span5", :label => _("Data store"), :label_size => "col-md-2", :disabled => true %>
 <% end %>
-<%= text_f f, :name, :class => "col-md-2", :label => _("Name"), :label_size => "col-md-2", :disabled => !new_host, :readonly => true %>
+<%= text_f f, :name, :class => "col-md-2", :label => _("Name"), :label_size => "col-md-2", :readonly => true %>
```

Comment:
I'll double check, but it should be fine. The name is ignored by VMWare anyway as you are not allowed to change a volume's name.